adapter_pipelines/submit.wdl

```diff
@@ -23,15 +24,17 @@ task get_metadata {
 
     get-analysis-metadata \
       --analysis_output_path ${analysis_output_path} \
-      --runtime_environment ${runtime_environment} \
+      --cromwell_url ${cromwell_url} \
       --use_caas ${use_caas}
   >>>
   runtime {
-    docker: "gcr.io/broad-dsde-mint-${runtime_environment}/cromwell-metadata:v0.19.0"
+    docker: "gcr.io/broad-dsde-mint-${runtime_environment}/cromwell-metadata:cw-update-to-new-bundle-structure"
   }
   output {
     File metadata = "metadata.json"
     String workflow_id = read_string("workflow_id.txt")
+    String adapter_workflow_id = read_string("adapter_workflow_id.txt")
```

Comment:
Can we remove this output file? It doesn't look like it's used anywhere else in the submit wdl.

Reply:
Sure, will remove it.